schemas/cloud.sql

```diff
@@ -211,7 +211,7 @@ DELIMITER ;
 
 -- -----------------------------------------------------------------------------
 -- Sites
-
+DROP TABLE IF EXISTS Sites;
```

Comment:
Good to have this inconsistency sorted out. :) Pretty much all the other tables have a `DROP TABLE`.